packages/utils/src/countries.ts

```diff
@@ -33,6 +33,13 @@ const matchCountry = (country: LocalizedCountry, query: string) => {
   )
 }
 
+// Iran, Cuba, North Korea, Sudan, Syria
+const bannedCountryCodeAlpha2 = ['IR', 'CU', 'KP', 'SD', 'SY']
+
+export const isCountryBanned = (countryCodeAlpha2: string) => {
+  return bannedCountryCodeAlpha2.indexOf(countryCodeAlpha2) >= 0
+}
+
```

Comment:
nit: was wondering if this should really be in utils, or specific only to Valora. Thoughts?

Reply:
Yeah, I think that makes more sense, moved!

Reply:
Cool ok in that case, we should use the [country feature flags](https://github.com/celo-org/celo-monorepo/blob/2ff4dfd0f15c7287995410c3006e75dd5bf06377/packages/mobile/src/flags.ts#L20-L26), happy to patch this myself to get that off your plate.

Reply:
Thanks for offering, but it looked like something quick and I think it was a good time for me to use these flags so that I get familiar with them to use in the future :)